### DISCUSSION TITLE
Can we put a static message on each PR?

### DISCUSSION THREAD

**Discussion by togfoxy** [togfoxy/MarsLander#225] [Ideas]
It would be good if github can display a message on each PR that gives the author a check list of the more common things to think about:

- 3 blank lines between functions
- locals declared as locals?
- camelCase for variables
- uppercase for globals

Etc. Not sure if that's possible.
Eyc

**Comment by Philbywhizz** [comment]
I think this is definately possible. I was only looking at this last night. You can setup pull request templates as well as issue templates. I'll have a look at setting this up tonight.

**Comment by Philbywhizz** [comment]
I've had some success, so now when you create a pull request it pre-popuates the description box with the following:

```
<!-- Some points to consider:                                               -->
<!--   * Have you updated the CHANGELOG.md for this pull request?           -->
<!--   * Check that your code follows the Style Guide located in the wiki.  -->

Changes proposed in this pull request.
-
-
-
```

If you want to change it then simply edit the .github/pull_request_template.md file in the repo

**Reply by togfoxy** [reply]
Fantastic!! I love how we're good at different things.  :)  I'll spend some time on this template next week. I don't want it to repeat the whole style guide (of course) but to target the principle offending items.

**Comment by Philbywhizz** [comment]
I've updated the code style guide with the 3 spaces between functions.
